### PR TITLE
quicksetup: fixes for wayland

### PR DESCRIPTION
**Pull request by 01micko**
- desensitize numlock
- bring back video section re #2686 #2688
- minor ui improvements with gtk2 and gtk3

### CODE REVIEW

File: woof-code/rootfs-skeleton/usr/sbin/quicksetup [hunk 1]
```diff
@@ -1173,6 +1187,9 @@ if [ "$SET_KEYBOARD" ];then
  fi
  if [ -f ~/.Xwaylandrc ]; then
   sed -i "s%^XKB_DEFAULT_LAYOUT=.*%XKB_DEFAULT_LAYOUT=$XKB_LAYOUT%" ~/.Xwaylandrc
+  if [ -f "$XDG_CONFIG_HOME/labwc/environment" ]; then
```
Comment (dimkr) [inline woof-code/rootfs-skeleton/usr/sbin/quicksetup]: I think this `if` should be outside of the previous one - we want it to be possible to have labwc without Cage and Xwayland

File: woof-code/rootfs-skeleton/usr/sbin/quicksetup [hunk 2]
```diff
@@ -949,10 +952,16 @@ if [ "$SET_XWIZARD" ];then
   #put up a button to launch xorgwizard...
   XDRIVERSUCCESS=$(report-video driver)
   if [ "$XDRIVERSUCCESS" ];then #precaution.
-    TT_xorg=$(gettext 'Xorg Video Wizard')
     B_xwiz=$(gettext 'Run Video Wizard')
-    XMSGX=$(gettext "The <b>${XDRIVERSUCCESS}</b> video driver is currently being used. Ok, if you need to adjust screen resolution or displacement, run the Video Wizard.")
-    XMSGRES=$(gettext "Current resolution:<b> $(report-video res) </b> ($(report-video depth) bit)")
+    if [ -z "$WAYLAND_DISPLAY" ];then
+		TT_xorg=$(gettext 'Xorg Video Wizard')
```
Comment (dimkr) [inline woof-code/rootfs-skeleton/usr/sbin/quicksetup:957]: Wayland Video Wizard?

**Review by dimkr — APPROVED** [review]

File: woof-code/rootfs-skeleton/usr/sbin/quicksetup [hunk 3]
```diff
@@ -1185,10 +1185,10 @@ if [ "$SET_KEYBOARD" ];then
  if [ -f /etc/X11/xorg.conf ]; then
   sed -i "s%.*Option.*XkbLayout.*%	Option      \"XkbLayout\" \"${XKB_LAYOUT}\"%" /etc/X11/xorg.conf
  fi
- if [ -f ~/.Xwaylandrc ]; then
-  sed -i "s%^XKB_DEFAULT_LAYOUT=.*%XKB_DEFAULT_LAYOUT=$XKB_LAYOUT%" ~/.Xwaylandrc
-  if [ -f "$XDG_CONFIG_HOME/labwc/environment" ]; then
-    sed -i "s%^XKB_DEFAULT_LAYOUT=.*%XKB_DEFAULT_LAYOUT=$XKB_LAYOUT%" $XDG_CONFIG_HOME/labwc/environment
+ if [ -f "$XDG_CONFIG_HOME/labwc/environment" ]; then
```
Comment (dimkr) [inline woof-code/rootfs-skeleton/usr/sbin/quicksetup:1188]: What I meant is,

```
if [ -f ~/.Xwaylandrc ]; then
  ...
fi

if [ -f "$XDG_CONFIG_HOME/labwc/environment" ]; then
  ...
fi
```

(no nesting)

~/.Xwaylandrc can exist if labwc is not present, and labwc can be present without Cage and .Xwaylandrc.

The way it is now, this change breaks quicksetup in dpup, with Cage and without labwc.

**Review by dimkr — CHANGES_REQUESTED** [review]
Keep .Xwaylandrc independent of labwc!

**Comment by 01micko** [comment]
I'll close and redo

**Comment by dimkr** [comment]
> I'll close and redo

You can always delete the branch locally (`git branch -D quicksetp3`), create it again (`git branch quicksetp3`), do your changes and force push (`git push -f quicksetp3`). This way, you don't have to close the PR and open again.

**Comment by 01micko** [comment]
Wondered how to force push. Thanks.

**Pull request closed without merging**